ocaml/driver/main_args.ml

```diff
@@ -919,6 +919,122 @@ let mk__ f =
   "<file>  Treat <file> as a file name (even if it starts with `-')"
 ;;
 
+let mk_flambda2_join_points f =
+  "-flambda2-join-points", Arg.Unit f, " Propagate information from incoming \
+    edges at a join point (Flambda 2 only)"
+;;
+
+let mk_no_flambda2_join_points f =
+  "-no-flambda2-join-points", Arg.Unit f, " Propagate information only from \
+    the fork point to a join point (Flambda 2 only)"
+;;
+
+let mk_flambda2_unbox_along_intra_function_control_flow f =
+  "-flambda2-unbox-along-intra-function-control-flow", Arg.Unit f,
+    " Pass values within a function as unboxed where possible (Flambda 2 only)"
+;;
+
+let mk_no_flambda2_unbox_along_intra_function_control_flow f =
+  "-no-flambda2-unbox-along-intra-function-control-flow", Arg.Unit f,
+    " Pass values within a function in their normal representation \
+      (Flambda 2 only)"
+;;
+
+let mk_flambda2_cse_depth f =
+  "-flambda2-cse-depth", Arg.Int f,
+    " Depth threshold for eager tracking of CSE equations (default 2) \
+      (Flambda 2 only)"
+;;
+
+let mk_flambda2_expert_code_id_and_symbol_scoping_checks f =
+  "-flambda2-expert-code-id-and-symbol-scoping-checks", Arg.Unit f,
+    " Perform checks on static scopes of code IDs and symbols during Un_cps \
+      (Flambda 2 only)"
+;;
+
+let mk_no_flambda2_expert_code_id_and_symbol_scoping_checks f =
+  "-no-flambda2-expert-code-id-and-symbol-scoping-checks", Arg.Unit f,
+    " Perform checks on static scopes of code IDs and symbols during Un_cps \
```

Comment:
This is the same description as for the "positive" flag.

Consider adding `(default)` to the description.